paleontographica-abteilung-b-palaeobotany-palaeophytology.csl

```diff
@@ -0,0 +1,246 @@
+    <id>http://www.zotero.org/styles/paleontographica-abteilung-b-palaeobotany-palaeophytology</id>
+    <link href="http://www.zotero.org/styles/paleontographica-abteilung-b-palaeobotany-palaeophytology" rel="self"/>
+    <link href="http://www.zotero.org/styles/annalen-des-naturhistorischen-museums-in-wien" rel="template"/>
+    <link href="https://www.schweizerbart.de/journals/palb/instructions" rel="documentation"/>
+    <author>
+      <name>Patrick O'Brien</name>
+      <email>obrienpat86@gmail.com</email>
+    </author>
+    <category citation-format="author-date"/>
+    <category field="science"/>
+    <issn>2194-900X</issn>
+    <eissn>2509-839X</eissn>
+    <summary>Style for Annalen des Naturhistorischen Museums in Wien</summary>
+    <updated>2017-04-04T12:34:19+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="en">
+    <terms>
+      <term name="editor" form="short">
+        <single>ed.</single>
+        <multiple>eds</multiple>
+      </term>
+      <term name="no date">undated</term>
+    </terms>
+  </locale>
+  <macro name="anon">
+    <choose>
+      <if variable="author editor translator" match="none">
+        <text term="anonymous" text-case="capitalize-first"/>
+      </if>
+    </choose>
+  </macro>
+  <macro name="container-contributors">
+    <choose>
+      <if type="chapter paper-conference" match="any">
+        <names variable="editor translator" delimiter=", ">
+          <name name-as-sort-order="all" and="symbol" font-variant="small-caps" initialize-with="." delimiter=", " delimiter-precedes-last="never"/>
+        </names>
+        <text macro="anon"/>
+      </if>
+    </choose>
+  </macro>
+  <macro name="author">
+    <names variable="author">
+      <name name-as-sort-order="all" and="symbol" sort-separator=", " initialize-with="." delimiter=", " delimiter-precedes-last="never" font-variant="small-caps"/>
+      <label form="short" prefix=" (" suffix=")" strip-periods="true"/>
+      <substitute>
+        <names variable="editor"/>
+        <names variable="translator"/>
+      </substitute>
+    </names>
+    <text macro="anon" font-variant="small-caps"/>
+  </macro>
+  <macro name="author-short">
+    <names variable="author">
+      <name form="short" and="symbol" delimiter=", " initialize-with=". " font-variant="small-caps"/>
+      <substitute>
+        <names variable="editor"/>
+        <names variable="translator"/>
+      </substitute>
+    </names>
+    <text macro="anon"/>
+  </macro>
+  <macro name="access">
+    <choose>
+      <if type="article-journal article-magazine article-newspaper bill book chapter graphic legal_case legislation motion_picture paper-conference report song" match="none">
+        <text variable="container-title" font-style="italic" suffix=". " prefix=" "/>
```

Comment:
remove the suffix here, the prefix for the DOI, and instead handle all of it with a group delimiter. (This, e.g. will print ". . " for items with container title and DOI.